<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1080</width>
     <height>611</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(16,16,16,255);
border-radius:20px;</string>
   </property>
   <widget class="QPushButton" name="AdminPanLogOff">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminPanLogOff{
background-color:rgba(255,125,0,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#AdminPanLogOff:pressed{
background-color:rgba(255,150,0,255);
}
</string>
    </property>
    <property name="text">
     <string> Log OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_all">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_all{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#btn_all:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AdminProBack">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#AdminProBack{
background-color:rgba(150,150,150,255);
color:rgb(255,255,255);
border-radius:20px;
}

QPushButton#AdminProBack:pressed{
background-color:rgba(200,200,200,255);
}
</string>
    </property>
    <property name="text">
     <string></string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_edit{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#btn_edit:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>90</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_add{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#btn_add:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLabel" name="LockImage">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(0,255,0,255);
align:center</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>871</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_refresh{
background-color:rgba(0,240,100,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#btn_refresh:pressed{
background-color:rgba(0,255,150,255);
}
</string>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>28</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="AdminTableSupp">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>871</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(150, 150, 150);</string>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Address ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Contact Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Phone</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Supplier Email</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Other Details</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="LockImage_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>50</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255,255,255,255);
align:center;
background-color: rgba(255, 255, 255, 100);</string>
    </property>
    <property name="text">
     <string>Suppliers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Leelawadee UI</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#Close{
background-color:rgba(240,0,0,255);
color:rgb(0,0,0);
border-radius:5px;
}

QPushButton#Close:pressed{
background-color:rgba(255,0,0,255);
}
</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
